examples/BUILD.gn

```diff
@@ -7,7 +7,7 @@ group("examples") {
 
   deps = [
     "//examples/demo_launcher",
-    "//examples/fitness",
+    #"//examples/fitness",
```

Comment:
k. Add a TODO or link to an issue or something so there's a breadcrumb trail explaining what this mysteriously commented-out line means. :-)